robotiq_modbus_rtu/src/robotiq_modbus_rtu/comModbusRtu.py

```diff
@@ -33,41 +33,58 @@
 # Copyright (c) 2012, Robotiq, Inc.
 # Revision $Id$
```

Comment:
There is a real mix of file naming conventions here. Can we clear this up somehow?